policy/cnspec_policy.proto

```diff
@@ -377,15 +484,28 @@ message ReportingJob {
   // This is replaced by the new spec, which now references cnquery
   map<string,DeprecatedV7_ScoringSpec> deprecated_v7_spec = 4;
   // ^^
+  // FIXME: DEPRECATED, remove in v10.0 vv
+  // This is replaced by the new type field, which now carries more info
+  bool deprecated_v8_is_data = 8;
+  // ^^
+
+  enum Type {
```

Comment:
I was thinking of keeping this embedded, since it's limited to ReportJobs. However, if we end up ever using it in other places, I agree and we would want to externalize it. I'm not sure about it